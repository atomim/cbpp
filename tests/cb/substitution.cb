#define MAKRO 100
a = b + MAKRO