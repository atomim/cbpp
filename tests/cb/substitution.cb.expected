// #define MAKRO 100
a = b + 100